vector/src/main/res/values-fr/strings.xml

```diff
@@ -849,7 +849,7 @@ Appareils inconnus :</string>
     <string name="tchap_invite_contacts_by_email">Inviter par mail</string>
     <string name="tchap_invite_contacts_to_room">Inviter au salon</string>
     <string name="tchap_room_invite_member_direct_chat">Nouvelle discussion</string>
-    <string name="tchap_invite_already_send_message">Vous avez déjà envoyé une invitation à %s</string>
+    <string name="tchap_invite_already_send_message">Une discussion avec %s existe déjà</string>
```

Comment:
You must keep the previous string "tchap_invite_already_send_message" (you may rename it tchap_invite_already_**sent**_message), and add a new one:  "tchap_invite_useless_message" (Une discussion avec %s existe déjà)